Source/FolioReaderPageIndicator.swift

```diff
@@ -80,9 +80,12 @@ class FolioReaderPageIndicator: UIView {
 		pagesLabel.textColor = isNight(UIColor(white: 1, alpha: 0.6), UIColor(white: 0, alpha: 0.9))
 	}
 
-    fileprivate func reloadViewWithPage(_ page: Int) {
+    fileprivate func reloadViewWithPage(_ page: Int, _ hidePageIndicator:Bool = readerConfig.hidePageIndicator) {
         let pagesRemaining = FolioReader.needsRTLChange ? totalPages-(totalPages-page+1) : totalPages-page
         
+        pagesLabel.isHidden = hidePageIndicator
```

Comment:
I don't think here is the best place to hide that, because you are inside the page indicator. 

Probably the best option is not to alloc and add the page indicator as subview, you can easily do that on `viewDidLoad` of the `FolioReaderCenter.swift` on line 166, where the page indicator is created. Just check for `hidePageIndicator` there.